### PR TITLE
[WIP] Proposed style constants implementation.

### PR DESCRIPTION
**Pull request by ImogenF**

### CODE REVIEW

File: packages/theme/src/index.ts [hunk 1]
```diff
@@ -1,184 +1,142 @@
...
-  navBackground: string
-  text: string
-  lightText: string
-  linkText: string
-  border: string
-  inputBorder: string
-  separator: string
+// Default constants
+const buttons = {
+  grey: {
```
[the remark this answers is not in the document]
Comment (micha-f) [reply packages/theme/src/index.ts]: That works 👍

File: packages/theme/src/index.ts [hunk 2]
```diff
@@ -1,184 +1,142 @@
...
+  blue: {
+    active: {
+      borderRadius: "2px",
+      backgroundColor: "#1499CE",
+      color: "#fff",
+    },
+  },
+  ghost: {
+    active: {
+      borderRadius: "2px",
```
Comment (fabien0102) [inline packages/theme/src/index.ts]: You can write `2` here (the basic number is in px by default)

File: packages/theme/src/index.ts [hunk 3]
```diff
@@ -1,184 +1,142 @@
...
-  border: string
-  inputBorder: string
-  separator: string
+// Default constants
+const buttons = {
+  grey: {
+    active: {
+      borderRadius: "2px",
+      backgroundColor: "#e2e2e2",
+      color: "#1499CE",
```
Comment (fabien0102) [inline packages/theme/src/index.ts]: Every colors over there should reference something into `fontColor` (to avoid any duplication) :wink:

Comment (micha-f) [reply packages/theme/src/index.ts]: @fabien0102 I disagree. We do not use some of these colors as font colors. I think it's perfectly fine to define the button colors without referencing anything. They are such basic elements of the UI.

**Comment by TejasQ** [comment]
Closing in favor of #492

**Pull request closed without merging**